.buildkite/scripts/bintray_upload.sh

```diff
@@ -40,9 +42,14 @@ echo "--- :habicat: Uploading core/hab to Bintray"
 # If we use Buildkite, we can potentially upload many different
 # platform artifacts to Bintray from a single platform (e.g., upload
 # Windows artifacts from Linux machines.)
-sudo hab pkg install core/hab --channel="${channel}"
+sudo "${hab_binary:?}" pkg install core/hab --channel="${channel}"
 
-hab_artifact=$(buildkite-agent meta-data get "hab-artifact")
+# TODO (SM): Should we specify Target for all of our platforms? 
```

Comment:
Probably so, particularly once we get the Windows builds working in Buildkite. Might as well take care of that now.